test/externalTests/zeppelin.sh

```diff
@@ -106,6 +106,10 @@ function zeppelin_test
     sed -i "s|it(\('other accounts cannot pause'\)|it.skip(\1|g" test/token/ERC721/presets/ERC721PresetMinterPauserAutoId.test.js
     sed -i "s|it(\('other accounts cannot unpause'\)|it.skip(\1|g" test/token/ERC721/presets/ERC721PresetMinterPauserAutoId.test.js
     sed -i "s|it(\('prevents initialization'\)|it.skip(\1|g" test/proxy/utils/Initializable.test.js
+    #TODO: Add more cases here to fix new errors
```

Comment:
This TODO is now done, isn't it? :)

Reply:
@cameel  Pushed changes to address this comment. Will squash commits once you approve.

Reply:
I meant that this line can just be removed.

Reply:
@matheusaaguiar  mentioned that we could add some explanation in his comment :  https://github.com/ethereum/solidity/pull/13092#issuecomment-1146380471

So, I added this extra comment. I can remove it if required.